src/dune_rules/melange_rules.ml

```diff
@@ -9,15 +9,23 @@ let make_js_name ~js_ext ~dst_dir m =
   let name = Melange.js_basename m ^ js_ext in
   Path.Build.relative dst_dir name
 
-let js_includes ~sctx ~target_dir ~requires_link ~scope ~js_ext =
+let js_includes ~loc ~sctx ~target_dir ~requires_link ~scope ~js_ext =
   let open Resolve.Memo.O in
   Command.Args.memo
     (Resolve.Memo.args
        (let* (libs : Lib.t list) = requires_link in
         let project = Scope.project scope in
         let deps_of_lib (lib : Lib.t) =
           let lib_name = Lib.name lib in
-          let lib = Lib.Local.of_lib_exn lib in
```

Comment:
There is another usage of this function in line 172 (now 180). Should it be replaced with the safe `of_lib` as well?